CHANGES.md

```diff
@@ -1,6 +1,20 @@
 Libplanet changelog
 ===================
 
+Version 0.5.2
+-------------
+
+To be released.
+
+### Backward-incompatible interface changes
+
+### Added interfaces
+
+### Behavioral changes
+
+### Bug fixes
```

Comment:
We don't need these sections other than bug fixes, because patch releases can have only bug fixes.

Reply:
Removed!